src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
Also, not sure how you'd otherwise reference `setTimeout` etc. in other modules if it wasn't a value returned from a module but just attached to it somehow.